OpenSim/Common/TRCFileAdapter.cpp

```diff
@@ -103,6 +120,10 @@ TRCFileAdapter::extendRead(const std::string& fileName) const {
     // Read the line containing column labels and fill up the column labels
     // container.
     auto column_labels = nextLine();
+    // for marker labels we do not need three columns per marker.
+    // remove the blank ones used in TRC due to tabbing
```

Comment:
It would be great if the comment could show a diagram or TRC snippet. The point here is that the TRC file looks like the following:

```
marker1<tab><tab><tab>marker2<tab><tab><tab>...
X1<tab>Y1<tab>Y2<tab>X2<tab>Y2<tab>Z2<tab>...
```

You're removing empty elements introduced by the tabs in the first of those rows, correct?